astro/src/content/docs/apis/identity-providers/_oauth-idp-request-body.mdx

```diff
@@ -64,8 +68,9 @@ import IdentityProviderLoginMethodRequestParameter from 'src/content/docs/apis/i
     This is an optional Application specific override for the top level <InlineField>scope</InlineField>.
 
     { props.idp_type === 'Google' && <>
-      <p class="text-green-600">Since 1.44.0</p>
-      The <InlineField>identityProvider.applicationConfiguration[applicationId].scope</InlineField> value may only be used when the <InlineField>identityProvider.loginMethod</InlineField> is set to <code>UseRedirect</code>. The <code>email</code>, <code>profile</code>, and <code>openid</code> scopes will always be requested for all other configurations.
+      <Aside type="version" title="Since 1.44.0">
+        The <InlineField>identityProvider.applicationConfiguration[applicationId].scope</InlineField> value may only be used when the <InlineField>identityProvider.loginMethod</InlineField> is set to <code>UseRedirect</code>. The <code>email</code>, <code>profile</code>, and <code>openid</code> scopes will always be requested for all other configurations.
```

Comment:
Can the `<code>` elements not be replaced with markdown backticks in this spot? I don't think it's worth breaking these all into separate components to get rid of the HTML tags, but I'm not sure of the limitations around that.

Reply:
Yeah, markdown doesn't work as soon as you enter a jsx expression `{props.idp_type === 'Google' && <> ... `.